src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx

```diff
@@ -98,28 +99,41 @@ const BrowserTrackConfig: FunctionComponent<BrowserTrackConfigProps> = (
     props.updateApplyToAll(!applyToAll);
   }, [applyToAll, updateApplyToAll]);
 
+  const checkboxStyles: any = {
```

Comment:
not `any`, please. Any is the last resort (and even then perhaps `unknown` is better, I don't know). Here the type is `{[key: string]: string}`. But it doesn't look like you even need to declare any type here; typescript should be capable of inferring it by itself.